tests/fetcher.spec.ts

```diff
@@ -0,0 +1,33 @@
+import * as fs from 'fs';
+import { homedir } from 'os'
+import {exec} from "child_process";
+
+import {test} from "@playwright/test";
+
+import {isracardFetcher} from "./isracardFetcher";
+import {maxFetcher} from "./maxFetcher";
+import {poalimFetcher} from "./poalimFetcher";
+
+const config = JSON.parse(fs.readFileSync(homedir() + '/config.json').toString())
+
+const {accounts} = config;
+
+const poalimAccounts = accounts.filter(account => account["type"] === 'poalim' && !account["skip_automation"]);
+const isracardAccounts = accounts.filter(account => account["type"] === 'isracard' && !account["skip_automation"]);
+const maxAccounts = accounts.filter(account => account["type"] === 'max' && !account["skip_automation"]);
```

Comment:
you can filter all skipAutomation accounts first and on that filter to each type